modules/admin/api/handler/cms_api_rels_admin_handler.e

```diff
@@ -0,0 +1,357 @@
+						      </div>
+						    </div>
+						    <div class="code">
+						      <h4>401 Unauthorized</h4>
+						      <div class="headers">
+						        <h5>Headers</h5>
+						        <ul>
+						          <li>WWW-Authenticate: indicates the Auth method (typically HTTP Basic)</li>
+						        </ul>
+						      </div>
+						    </div>
+						  </div>
+						</div>
+
+						</body>
+						</html>
+		]"
+
+	rel_api_role_update: STRING = "[
+						<!DOCTYPE html>
+						<html>
+						<head>
+							<link href="/rels.css" media="all" rel="stylesheet" type="text/css">
+						</head>
+						<body>
+						<h1 class="page-header">api-role-update <small>relation</small> </h1>
+			
+						<div class="well method"><h2>PUT</h2>
+							<p class="description">Update the current role</p>
+							<div class="request"><h3>Responses</h3>
+								<div class="headers">
+									<h4>Request Headers</h4>
+										<div class="type">
+											The request should have the Content-Type application/json
+										</div>
+								</div>
+							<div class="body">
+								<h4>Body</h4>
+									<div class="required">
+										<h5>Required properties</h5>
+										<ul>
+											<li><stong>name</strong> : "The string containing the new name of the role"
+										</li>
+										</ul>
+											<h5>Example</h5>
+										<pre>
+										"{
+											"name": "users"
+										}"
+										</pre>
+										</div>
+									</div>
+							</div>
+
+								<div class="code">
+									<h4>204 No Content</h4>
+								</div>
+						    </div>
+						    <div class="code">
+						      <h4>401 Unauthorized</h4>
+						      <div class="headers">
+						        <h5>Headers</h5>
+						        <ul>
+						          <li>WWW-Authenticate: indicates the Auth method (typically HTTP Basic)</li>
+						        </ul>
+						      </div>
+						    </div>
+						  </div>
+						</div>
+						</div>
+					</body>
+					</html>
+		]"
+
+	rel_api_role_delete: STRING = "[
+						<!DOCTYPE html>
+						<html>
+						<head>
+							<link href="/rels.css" media="all" rel="stylesheet" type="text/css">
+						</head>
+						<body>
+						<h1 class="page-header">api-role-delte <small>relation</small> </h1>
+			
+						<div class="well method"><h2>DELETE</h2>
+							<p class="description">Delete the current role</p>
+			
+							<div class="response"><h3>Responses</h3>
+								<div class="code">
+									<h4>204 No Content</h4>
+								</div>
+							</div>
+						    <div class="code">
+						      <h4>401 Unauthorized</h4>
+						      <div class="headers">
+						        <h5>Headers</h5>
+						        <ul>
+						          <li>WWW-Authenticate: indicates the Auth method (typically HTTP Basic)</li>
+						        </ul>
+						      </div>
+						    </div>
+						  </div>
+						</div>
+						</div>
+					</body>
+					</html>
+		]"
+
+feature -- {NONE}
+
+	compute_response_get (req: WSF_REQUEST; res: WSF_RESPONSE; output: STRING; media_type: STRING)
```

Comment:
I dislike the name , and no comment

from the name, I could think it is .. prepare Current response.
But no .. since this is "sending" the response ...